tests/Cms/Collections/fixtures/collections/options_with_default.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+use Kirby\Cms\App;
+
+return function (int $a, int $b, int $c, App $kirby) {
```

Comment:
If I understand the code correctly, like in controllers, we don't need to rely on the order of arguments. Would be good to have one test where the Closure has the arguments in a different order than they are passed in the test to the callback.